pancancer_utilities/data_utilities.py

```diff
@@ -252,4 +247,41 @@ def summarize_results(results, gene, holdout_cancer_type, signal, z_dim,
 
     return metrics_out_, roc_df_, pr_df_
 
+def subset_by_mad(X_train_df, X_test_df, gene_features, subset_mad_genes, verbose=False):
+    """Subset features by mean absolute deviation.
+
+    Takes the top subset_mad_genes genes (sorted in descending order),
+    calculated on the training set.
+
+    Arguments
+    ---------
+    X_train_df: training data, samples x genes
+    X_test_df: test data, samples x genes
+    gene_features: numpy array of gene features
```

Comment:
Looks like `gene_features` is an array of bool, right? It might be good to add that here.

I understand you have different features in `X_train_df`, and this parameter indicates which ones are genes.